Content.Server/Shuttles/Systems/ArrivalsSystem.cs

```diff
@@ -154,6 +154,14 @@ public override void Shutdown()
 
     private void OnArrivalsFTL(EntityUid shuttleUid, ArrivalsShuttleComponent component, ref FTLStartedEvent args)
     {
+        if (!TryGetArrivals(out EntityUid arrivals) || !TryComp<TransformComponent>(arrivals, out var arrivalsXform))
+            return;
+
+        var arrivalsMapUid = Transform(arrivals).MapUid;
```

Comment:
You already have arrivals xform comp above so just re-use it.

Reply:
Oh, I was supposed to remove the arrivals trycomp